modules/Bio/EnsEMBL/Utils/IO/GFFSerializer.pm

```diff
@@ -260,8 +275,8 @@ sub print_feature {
 		}
             }
             else {
-                if (defined $summary{$attribute}) { 
-                  $row .= $attribute."=".uri_escape($summary{$attribute},'\t\n\r;=%&,'); 
+                if (defined $summary{$attribute}) {
+                  $row .= $attribute."=".uri_escape($summary{$attribute},'\t\n\r;=%&,');
```

Comment:
Seeing as you have already stripped the trailing whitespace here, maybe you could add some _inline_ whitespace for readability as well?

Reply:
done